src/toast.toasts.styles.ts

```diff
@@ -0,0 +1,127 @@
+import { css } from 'lit';
+import visuallyHidden from './styles/visually-hidden.js';
+
```

Comment:
Mostly a copy-pasta of [`src/toasts.styles.ts`](https://github.com/CrowdStrike/glide-core/pull/847/files#diff-ad1bb79ebc055d46baa175be4da7479b1ad1efc1fd886bac2020456e544525f1) and [`src/toasts.toast.styles.ts`](https://github.com/CrowdStrike/glide-core/pull/847/files#diff-2e2e9ba9b0b6e5230bf8c7274d9a26a33fbeef7b5a833f651c6452ee2cbecfac).